### DISCUSSION TITLE
can't download teleport

### DISCUSSION THREAD

**Discussion by mstein-IDEX** [gravitational/teleport#30123] [Q&A] [answered]
I get the following error running the following command:

```
sudo bash -c "$(curl -fsSL https://citeqw.teleport.sh/scripts/1f03d7cwerwerwer56787cf996488170bef5a0675/install-node.sh)"

2023-08-07 07:41:29 CDT [teleport-installer] Installing repo for distro debian.
Hit:1 http://deb.debian.org/debian bullseye InRelease
Get:2 http://security.debian.org/debian-security bullseye-security InRelease [48.4 kB]
Hit:3 http://deb.debian.org/debian bullseye-updates InRelease                 
Get:4 https://apt.releases.teleport.dev/debian bullseye InRelease [131 kB]    
Hit:5 http://archive.raspberrypi.org/debian bullseye InRelease                
Err:4 https://apt.releases.teleport.dev/debian bullseye InRelease
  Unknown error executing apt-key
Reading package lists... Done
W: GPG error: https://apt.releases.teleport.dev/debian bullseye InRelease: Unknown error executing apt-key

N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'https://apt.releases.teleport.dev/debian bullseye InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
root@citpi1:/# 
```

thanks

**Comment by webvictim** [comment]
I tried this on fresh Debian 11 (bullseye) servers - both an EC2 instance and a Docker container. It worked fine both times. This may have been a temporary issue with the CDN, or, more likely, you don't have a necessary dependency for the script installed.

If you try to install the Debian repo yourself by running each of these commands below, what happens?

```console
sudo curl https://apt.releases.teleport.dev/gpg -o /usr/share/keyrings/teleport-archive-keyring.asc
source /etc/os-release
echo "deb [signed-by=/usr/share/keyrings/teleport-archive-keyring.asc] https://apt.releases.teleport.dev/${ID?} ${VERSION_CODENAME?} stable/v13" | sudo tee /etc/apt/sources.list.d/teleport.list > /dev/null
sudo apt-get update
sudo apt-get install teleport-ent
```

**Reply by mstein-IDEX** [reply]
OK, i got teleport installed, it seems installing "teleport-ent-updater" is the issue.

Now that I have Teleport installed how do I associate it with my Teleport cloud?

thanks

**Reply by webvictim** [reply, marked as answer]
Replace `example.teleport.sh` with your own Teleport tenant name in these commands.

## From your local machine

0) Make sure you have `tsh` and `tctl` installed locally - download the relevant package from https://goteleport.com/docs/choose-an-edition/teleport-cloud/downloads/ if not

1) Log into your Cloud instance, replacing `<email>` with your Teleport Cloud username:
```
tsh login --proxy=example.teleport.sh --user=<email>
```

2) Generate a token for the SSH service:
```
tctl tokens add --type=node --ttl=30m
```

Copy the token outputted, you'll need it in step 3.

## On your Teleport agent machine

3) Run this command to generate a config file:
```
sudo teleport node configure --proxy=example.teleport.sh:443 --token=<token from step 2> --output=file:///etc/teleport.yaml
```

4) Start Teleport:
```
sudo systemctl start teleport
```

Your node should join your Teleport Cloud cluster.

View the Teleport process logs with `sudo journalctl -u teleport` if you don't see it come online.

**Reply by mstein-IDEX** [reply]
All good now.
